README.md

```diff
@@ -0,0 +1,85 @@
+# doctrine-sortable
+
+Sooner or later you will have to implement sorting of your entities.
+For example categories, products on main page and so on. And why you should do this by yourself when everything you have to do is to copy & paste?
+
+
+Installation
+-----------
+
+The best way to install Kdyby/Doctrine is using [Composer](http://getcomposer.org/):
+
+```sh
+$ composer require librette/doctrine-sortable
+```
+
+and enable librette extension in your `config.neon
+````yml
+extensions:
+	# add this line at the end of your extensions list
+	librette.doctrine.sortable: Librette\Doctrine\Sortable\DI;
+```
+
+Simplest entity
+---------------
+
+
+```php
+namespace App;
+
+use Kdyby\Doctrine\Entities\BaseEntity;
+use Librette\Doctrine\Sortable\ISortable;
+use Librette\Doctrine\Sortable\TSortable;
+
+/**
+ * @ORM\Entity
+ */
+class Article extends BaseEntity implements ISortable
+{
+
+	use TSortable;
+
+	/**
+	 * @ORM\Id
+	 * @ORM\Column(type="integer")
+	 * @ORM\GeneratedValue
+	 */
+	protected $id;
+
+	/**
+	 * @ORM\Column(type="integer")
+	 */
+	protected $position;
```

Comment:
both this property and getPosition method are useless, it is already in the trait